public/js/client.js

```diff
@@ -54,7 +54,11 @@ ready(function(){
         xhr.send(params);
     }
 
+<<<<<<< HEAD
     // Detect #LOGIN button click, Make POST request TO THE SERVER
+=======
+    // POST TO THE SERVER
+>>>>>>> 701adf1 (First Push)
```

Comment:
Whoops, forgot to resolve this conflict.

```suggestion
    // Detect #LOGIN button click, Make POST request TO THE SERVER
```